staff/models/batch.py

```diff
@@ -16,6 +16,10 @@ class BasicsBatchManager(models.Manager):
     def get_queryset(self):
         return super().get_queryset().filter(deleted_at__isnull=True, course=Course.objects.get(name=Course.CODING_BASICS))
 
+class SWEFundamentalsBatchManager(models.Manager):
```

Comment:
Could we use camel case even for acronyms, like `SweFundamentalsBatchManager`? I find that makes reading camel case easier! https://google.github.io/styleguide/jsguide.html#naming-camel-case-defined

Reply:
let's follow python styling conventions rather than js styling conventions since this is a Django proj https://legacy.python.org/dev/peps/pep-0008/#descriptive-naming-styles